syndeo-template/src/main/java/com/google/cloud/syndeo/common/ProviderUtil.java

```diff
@@ -117,11 +119,19 @@ public static PCollectionRowTuple applyConfigs(
       Collection<TransformSpec> specs, PCollectionRowTuple tuple) {
     for (TransformSpec spec : specs) {
       SchemaTransform transform = spec.provider.from(spec.configuration);
+
+      if (tuple.getAll().containsKey(ERRORS_TAG)) {
+        // TODO(pabloem): Deal with errors.
+      }
       // We know we only deal with transforms with either 0 or 1 input so we know how to connect the
       // collections. To sanity check we should confirm the output collections match expected.
-      if (tuple.getAll().size() == 1) {
+      if (tuple.getAll().size() == 1 || tuple.getAll().containsKey(OUTPUT_TAG)) {
```

Comment:
Not sure what the tag check buys you? If the size is either 0 or 1 this will always match no matter whether or not the tuple contains OUTPUT_TAG.

Reply:
the size could be more than 1, right? "errors", "output" for example.